marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/provenance/pruning.sjs

```diff
@@ -26,7 +26,7 @@ function validatePruneRequest({ retainDuration, batchSize = 100 }) {
 function pruneProvenance(pruneRequest, endpointState) {
     xdmp.securityAssert("http://marklogic.com/data-hub/privileges/prune-provenance", "execute");
     // update with validated request properties
-    Object.assign(pruneRequest, validatePruneRequest(pruneRequest));
+    pruneRequest = validatePruneRequest(pruneRequest);
```

Comment:
I think this module should just be "prov-lib.sjs", and likely the future home of all the logic in "prov.sjs" today. That makes it consistent with e.g. "entity-lib.sjs".